docs/GUIDE.md

```diff
@@ -0,0 +1,229 @@
+# A Warrior's Guide to Thunderdome
+
+Thunderdome is a fun way to perform agile scrum planning. See Wikipedia.
+
+As a new warrior in this realm, let this be your guide to Battles. First, we need to know who you are.
+
+## Register (optional)
+
+Create a new account, or join as guest.
+
+Having an account lets you save your battles and more.
+
+- Name
+This will be visible to others.
+- Email (for account)
+- Password (for account)
+Use a strong password. Type it again to confirm
+
+You will receive an email to confirm your new account.
+
+## Login
+
+Use the email/password you created when registering.
+
+- Email
+- Password
+
+_OIDC Providers coming soon._
+
+### Password Retrieval
+
+Forgot your password? Thunderdome can send a link to your email.
+
+## Profile
+
+Warrior, it is all about you! Control your Thunderdome experience.
+
+### Details
+
+- Name
+This will be visible to others.
+- Email
+Update your account email, or enter one if you are a guest.
+- Country (optional)
+- Locale, default: English
+8 locales to choose from.
+- Company (optional)
+- Job Title (optional)
+- Theme, default: auto
+If you prefer a darker or lighter interface, you may choose it here.
+- Option: Enable Battle Notification, default: true
+- Avatar, default: robohash
+Several others to choose from, pick your flavor; mp, identicon, monsterid, wavatar, retro.
+
+### API Access
+
+Create an API key to integrate Thunderdome with your tools. 
+
+See API Documentation here [Thunderdome API Docs](https://thunderdome.dev/swagger/index.html)
+
+### Jira Integration (premium only)
+
+Integrate directly with your team's backlog.
+
+_Other integrations coming soon._ 
+
+### Delete Account
+
+This is the Thunderdome, but you are free to leave. We will erase all data about you.
```

Comment:
`This is permanent: All poker sessions, retros, story maps, and orgs/teams directly owned by your account will also be deleted.`

Reply:
```suggestion
This is the Thunderdome, but you are free to leave. We will erase all data about you.

**This is permanent:** All poker sessions, retros, story maps, and orgs/teams directly owned by your account will also be deleted.
```